package.json

```diff
@@ -24,6 +35,7 @@
     "eslint-plugin-react-hooks": "^4.6.0",
     "eslint-plugin-react-refresh": "^0.4.5",
     "postcss": "^8.4.35",
+    "sass": "^1.71.1",
```

Comment:
Do we use sass in our project?

Reply:
yes, we do. should we use it, shouldn't we?

Reply:
if we dont use it, we will uninstall it right?

Reply:
@NhatNguyen1502 : yes, please remove it

Reply:
done